src/main/java/seedu/address/logic/Logic.java

```diff
@@ -30,11 +30,15 @@
      */
     ReadOnlyAddressBook getAddressBook();
 
-    /** Returns an unmodifiable view of the filtered list of persons */
-    ObservableList<Person> getFilteredPersonList();
+    /**
+     * Returns an unmodifiable view of the filtered list of passengers.
```

Comment:
Should the javadocs be represented like this or would a single line as per the original document be better?

Reply:
closing as upon discussion, we agreed that it's alright as checkstyle tests have passed it